support/Python/Schedule.py

```diff
@@ -0,0 +1,779 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+import re
+import shutil
+import subprocess
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Optional, Sequence, Union
+
+import click
+import jinja2
+import jinja2.meta
+import yaml
+
+logger = logging.getLogger(__name__)
+
+default_submit_script_template = Path(__file__).parent / 'Submit.sh'
```

Comment:
Yep it's next to the `Schedule.py` file in `support/`. That's were CMake puts the machine-specific submit script template. It works with `PY_DEV_MODE` because the `.parent` is purely lexical (doesn't resolve symlinks). I'm open to better ideas how to copy around submit scripts!

Reply:
Ok this seems fine then